src/context/types.ts

```
@@ -0,0 +1,30 @@
+    id: string;
+    type: 'group' | 'private';
+
+    send(content: string): Promise<void>;
+}
+
+export interface Content {
+    body: string;
+    media: Media | undefined;
+}
+
+export interface Media {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)